app/controllers/admin/plugins_controller.rb

```diff
@@ -8,4 +8,13 @@ def index
       root: "plugins",
     )
   end
+
+  private
+
+  def preload_additional_json
```

Comment:
Don't want to do this on every single StaffController because it's "need to know" info. I think the repetition is acceptable...for now at least.

Reply:
![](https://media1.giphy.com/media/NCjISbEPFxm48/giphy.gif?cid=ecf05e470zo8m1bd4pwl4348ayuw1g5kxxpp34wzzccxrubu&ep=v1_gifs_search&rid=giphy.gif&ct=g)